packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -1,135 +1,52 @@
-
-This design is intentionally minimal while still being useful.
-Features not needed for minimal strongly typed application use or the JSON and XML domain schema are not included.
-
-## Schema Representation
-
-The MVP should include a simple schema system, usable as an MVP for both `stored schema` and `view schema`. One possible such schema system is included in [schema.ts](./schema.ts).
-
-## Use as `Stored Schema`
-
--   Writers: When inserting new content and updating modified nodes, shared-tree can check them against the schema in the document.
-
-    Each node has a type that can reference a schema. For stored nodes, this is a stored schema and it must exist in the document.
-    If you insert something into a document that does not currently have schemas for its types, you also have to include the schema as part of the edit.
-
-    For example, if you add support for tables to whiteboards, old documents will not have this schema, so part of inserting a table will require updating the schema to allow them (if the document is explicit about which item types are allowed), as well as adding the table schema itself.
-
-    This is necessary because otherwise adding a new type could break existing data which might not even be downloaded on the current client.
-    In [schema.ts](./schema.ts), `TreeSchema` and `FieldSchema` can be added to an edit op to be used as `stored schema`.
-
-    Note: It's possible to add a schema that is compatible with all possible data (assuming the children themselves are compatible with their own types).
-    Applications which wish to rely entirely on schema-on-read for some or all of their data can use this pattern for all `stored schema` and only use their actual developer authored schema as `view schema`.
--   Change application: changes can conflict if they violate the schema (or maybe only check at the end of a transaction? Or at special marked places and at end?).
-
-    Change application could (someday) adjust behavior based on the schema (ex: provide set semantics to a sequence) or have schema specific edits but the initial version will just detect violations and mark them as conflicted.
-
-### Ways to update existing schema:
-
--   Could support op that changes a schema in a way where all data that was accepted by the old schema is accepted by the new one (ex: add optional fields (if compatible with extra fields), move required field into extraFields, add a type to a field).
--   Could even do things like apply a change to all nodes with a specific type to enable edits which modify a schema and update data atomically (ex: alter table like from sql). This does not work with partial checkouts well.
-
-## Use as `View Schema`
+This document covers our chosen design from the design space defined by [Stored and View Schema Options](./Stored%20and%20View%20Schema%20Options.md).
+
+```mermaid
+graph LR;
```

Comment:
I'm not quite sure what all of these arrows mean. Could you add labels? Particularly between relationships like container and application which I imagine are different from view schema and field schema map.

Reply:
All these arrows are pretty much the same. I added a paragraph introducing this diagram to help with that.